DISPLAY "(null)" invalid; disabling X11 forwarding
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcuda.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcurand.so locally
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
E tensorflow/stream_executor/cuda/cuda_driver.cc:491] failed call to cuInit: CUDA_ERROR_NO_DEVICE
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:153] retrieving CUDA diagnostic information for host: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:160] hostname: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:185] libcuda reported version is: Not found: was unable to find libcuda.so DSO loaded into this program
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:347] driver version file contents: """NVRM version: NVIDIA UNIX x86_64 Kernel Module  367.27  Thu Jun  9 18:53:27 PDT 2016
GCC version:  gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.3) 
"""
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:189] kernel reported version is: 367.27.0
I tensorflow/core/common_runtime/gpu/gpu_init.cc:81] No GPU devices available on machine.
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, raiders2:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, localhost:2210, raiders3:2211, raiders3:2212, raiders8:2213, raiders8:2214, raiders8:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2210
INFO:tensorflow:Learning rate: 0.002500, momentum: 0.900000
INFO:tensorflow:Sync mode!!!!!!
INFO:tensorflow:Compute groups : 2
INFO:tensorflow:SyncReplicas enabled: replicas_to_aggregate=16; total_num_replicas=16
INFO:tensorflow:Num nodes per compute groups: 8
INFO:tensorflow:Tokens per step: 8
INFO:tensorflow:2016-07-27 03:39:56.145597 Supervisor
E0727 03:40:12.345066760   22096 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2205': timeout occurred
E0727 03:40:12.345143070   22096 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2206': timeout occurred
E0727 03:40:12.345161515   22096 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2208': timeout occurred
E0727 03:40:12.345176286   22096 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2207': timeout occurred
E0727 03:40:15.023909471   22096 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2206': timeout occurred
E0727 03:40:15.333163915   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:40:15.333217303   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:40:15.333232997   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:40:15.333269299   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:40:18.333108142   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:40:18.333142454   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:40:18.333155631   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:40:21.333117095   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:40:21.333150862   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:40:21.333163582   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:40:24.333114165   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:40:30.333057245   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:40:30.333090422   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:40:33.333051539   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:40:33.333080793   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:40:42.357105283   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:40:48.357063319   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:40:48.357108476   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:40:51.357028173   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:41:05.889039114   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:41:08.889019591   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:41:11.889032767   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:41:15.141016164   22843 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
run-expr-32node.sh: line 23: 22076 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
